packages/cacti-ledger-browser/package.json

```diff
@@ -58,7 +58,7 @@
     "@supabase/supabase-js": "1.35.6",
     "@tanstack/react-query": "5.29.2",
     "apexcharts": "3.45.2",
-    "axios": "1.7.7",
+    "axios": "1.7.4",
```

Comment:
It's again downgrading axios version here too.

Reply:
@sandeepnRES I usually just do a search and replace in the IDE and then do a commit message amend so that the lines that are longer than a 100 characters are not.